gems/smithy/spec/interfaces/client/shapes_spec.rb

```diff
@@ -11,188 +11,431 @@
 
   let(:fixture) { JSON.load_file(File.expand_path('../../fixtures/shapes/model.json', __dir__.to_s)) }
   let(:shapes_module) { Smithy::Client::Shapes }
+
   subject { ShapeService::Shapes }
 
   it 'generates a shapes module' do
     expect(subject).to be_a(Module)
   end
 
-  context 'generated shapes' do
-    context 'string' do
-      let(:expected_shape) { fixture['shapes'].select { |_, s| s['type'] == 'string' } }
+  context 'blob' do
+    it 'is an instance of BlobShape' do
+      expect(subject::Blob).to be_a(shapes_module::BlobShape)
+    end
 
-      it 'is an instance of StringShape' do
-        expect(subject::String).to be_a(shapes_module::StringShape)
-      end
+    it 'has an id' do
+      id, = fixture['shapes'].find { |_, s| s['type'] == 'blob' }
+      expect(subject::Blob.id).to eq(id)
+    end
 
-      it 'has an id' do
-        expect(subject::String.id).to eq(expected_shape.keys.first)
-      end
+    it 'has traits' do
+      _, shape = fixture['shapes'].find { |_, s| s['type'] == 'blob' }
+      expect(subject::Blob.traits).to eq(shape['traits'])
+    end
+  end
 
-      it 'has traits and does not contain omitted traits' do
-        expected_traits =
-          expected_shape
-          .values
-          .first['traits']
-          .reject { |k, _| k.include?('documentation') }
```

Comment:
Are we still testing that some traits are omitted?  I didn't see this same test preserved, but its also a lot of new tests so may have missed it.

Reply:
Including them won't break behavior, but I think its an important part of the code generation logic and so that behavior should get tested somewhere.